llvm/lib/Target/X86/AsmParser/X86AsmParser.cpp

```diff
@@ -3784,6 +3795,61 @@ bool X86AsmParser::processInstruction(MCInst &Inst, const OperandVector &Ops) {
     Inst.setOpcode(X86::INT3);
     return true;
   }
+  // `{evex} cmp <>, <>` is alias of `ccmpt {dfv=} <>, <>`
+#define FROM_TO(FROM, TO)                                                      \
+  case X86::FROM: {                                                            \
+    return replaceWithCCMP(X86::TO);                                           \
+  }
```

Comment:
Nit: This can be one line: `case X86::FROM: return replaceWithCCMP(X86::TO)`

Reply:
[ee1d9e4](https://github.com/llvm/llvm-project/pull/99277/commits/ee1d9e48da1e8662e974b7dedecd386ebddd94c8)